mycelo/genesis/base_config.go

```diff
@@ -42,7 +42,7 @@ func BaseConfig() *Config {
 			Symbol:                      "cUSD",
 			Decimals:                    18,
 			Rate:                        fixed("1"),
-			InflationFactorUpdatePeriod: bigInt(2 * Year),
+			InflationFactorUpdatePeriod: bigInt(1 * Week),
```

Comment:
why 1 week? i understand that we want to turnf inflation off, so 2 years is far away in the future

Reply:
I'll move this to the load test template, it's to more closely replicate mainnet conditions (so gas usage per tx will match)

Reply:
are you sure that's the case? my guess is taht inflation is "off" o mainnet...

Reply:
It's not quite off, but `updatePeriod: 1 year, 6 months, 2 days, 12 hours` on mainnet. I'll switch this back.